test/src/xquery/fn.xql

```diff
@@ -21,4 +41,17 @@ declare
 function fnt:normalize-unicode($normalization-form as xs:string) {
     count(string-to-codepoints(normalize-unicode($fnt:composed, $normalization-form))),
     count(string-to-codepoints(normalize-unicode($fnt:decomposed, $normalization-form)))
-};
\ No newline at end of file
+};
+
+declare
```

Comment:
There was no test before. I added the tests that I needed. Feel free to send a PR with more tests if you want them.